packages/maskbook/src/components/shared/NetworkIcon.tsx

```diff
@@ -15,22 +15,20 @@ export interface NetworkIconProps extends withClasses<'icon'> {
     networkType?: NetworkType
 }
 
-const EthereumIcon = new URL('../../resources/wallet-network-icon/ethereum.png', import.meta.url).toString()
-const BinanceIcon = new URL('../../resources/wallet-network-icon/binance.png', import.meta.url).toString()
-const PolygonIcon = new URL('../../resources/wallet-network-icon/polygon.png', import.meta.url).toString()
+const icons: EnumRecord<NetworkType, string> = {
```

Comment:
Please use `Record` and remove `EnumRecord`. Thanks. (These two types are identical.)

Reply:
REMOVED.